CHANGELOG.md

```diff
@@ -55,6 +55,8 @@ and is expected to change will be tagged interchangably as "experimental" or
 
 - [BUGFIX] agentctl will no longer leak WAL resources when retrieving WAL stats. (@rfratto)
 
+- [BUGFIX] Include Windows Installer when building for release (@mattdurham)
```

Comment:
Can you move everything in main to a header for 0.14.0-rc.0 and add this to a new master/unerged section?